ui/raidboss/data/06-ew/dungeon/another_sildihn_subterrane.txt

```diff
@@ -0,0 +1,225 @@
+233.0 "Bracing Duster?" #sync / 1[56]:[^:]*:Silken Puff:7769:/
+233.0 "Fizzling Duster?" #sync / 1[56]:[^:]*:Silken Puff:776A:/
+244.2 "Dust Bluster" sync / 1[56]:[^:]*:Silkie:776C:/
+249.6 "Bracing Suds" sync / 1[56]:[^:]*:Silkie:7757:/
+255.7 "Fresh Puff" sync / 1[56]:[^:]*:Silkie:7766:/
+262.3 "--sync--" sync / 1[56]:[^:]*:Silkie:(7751|7752):/
+263.8 "Squeaky Clean 1" sync / 1[56]:[^:]*:Silkie:7753:/
+265.5 "Squeaky Clean 2" sync / 1[56]:[^:]*:Silkie:7754:/
+266.9 "Puff and Tumble x4" duration 0.7 #sync / 1[56]:[^:]*:Silken Puff:7771:/
+266.9 "Squeaky Clean 3" sync / 1[56]:[^:]*:Silkie:(7755|7756):/
+276.0 "Soaping Spree" sync / 1[56]:[^:]*:Silkie:7767:/
+276.0 "Chilling Duster x4?" #sync / 1[56]:[^:]*:Silken Puff:7768:/
+276.0 "Bracing Duster" #sync / 1[56]:[^:]*:Silkie:7769:/
+# 276.0 "Fizzling Duster?" #sync / 1[56]:[^:]*:Silken Puff:776A:/ # Possible here, but means a mechanic failed
+# 276.0 "Bracing Duster?" sync / 1[56]:[^:]*:Silken Puff:7769:/ # Possible here, but means a mechanic failed
+283.1 "Total Wash" sync / 1[56]:[^:]*:Silkie:7750:/
+292.3 "Bracing Suds / Chilling Suds / Fizzling Suds" sync / 1[56]:[^:]*:Silkie:(7757|7758|7759):/
+294.4 "--sync--" sync / 1[56]:[^:]*:Silkie:79FB:/
+299.4 "Slippery Soap" sync / 1[56]:[^:]*:Silkie:775E:/
+303.5 "Bracing Duster / Chilling Duster / Fizzling Duster" sync / 1[56]:[^:]*:Silkie:(7764|7763|7765):/
+303.8 "Soapsud Static?" #sync / 1[56]:[^:]*:Silken Puff:77ED:/
+309.0 "Bracing Suds / Chilling Suds / Fizzling Suds" sync / 1[56]:[^:]*:Silkie:(7757|7758|7759):/
+311.1 "--sync--" sync / 1[56]:[^:]*:Silkie:79FB:/
+316.1 "Slippery Soap" sync / 1[56]:[^:]*:Silkie:775E:/
+320.0 "Bracing Duster / Chilling Duster / Fizzling Duster" sync / 1[56]:[^:]*:Silkie:(7764|7763|7765):/
+322.9 "--sync--" sync / 14:[^:]*:Silkie:79E0:/ window 15,5
+332.9 "Total Wash (enrage)" sync / 1[56]:[^:]*:Silkie:79E0:/
+338.1 "Total Wash (enrage)" sync / 1[56]:[^:]*:Silkie:79F9:/
+
+#~~~~~~~~~~~~~~~~~~~~~~#
+# Gladiator of Sil'dih #
+#~~~~~~~~~~~~~~~~~~~~~~#
+
+# -ii 766F 7670 7676 7677 77BF 7660
+
+1000.0 "--sync--" sync / 00:0839::The Trial of Might will be sealed off/ window 1000,1
+1006.8 "--sync--" sync / 14:[^:]*:Gladiator of Sil'dih:7671:/ window 1007.3,5
+1011.5 "Flash of Steel" sync / 1[56]:[^:]*:Gladiator of Sil'dih:7671:/
+1023.6 "Specter of Might" sync / 1[56]:[^:]*:Gladiator of Sil'dih:7673:/
+1038.3 "Rush of Might 1" sync / 1[56]:[^:]*:Gladiator of Sil'dih:765B:/
+1040.4 "Rush of Might 2" sync / 1[56]:[^:]*:Gladiator of Sil'dih:765C:/
+1042.8 "Specter of Might" sync / 1[56]:[^:]*:Gladiator of Sil'dih:7673:/
+1057.5 "Rush of Might 1" sync / 1[56]:[^:]*:Gladiator of Sil'dih:765B:/
+1059.6 "Rush of Might 2" sync / 1[56]:[^:]*:Gladiator of Sil'dih:765C:/
+1062.0 "--sync--" sync / 1[56]:[^:]*:Gladiator of Sil'dih:6854:/
+1067.0 "Sculptor's Passion" sync / 1[56]:[^:]*:Gladiator of Sil'dih:766C:/
+1067.3 "Sculptor's Passion" sync / 1[56]:[^:]*:Gladiator of Sil'dih:79F3:/
```

Comment:
I'm guessing that `79F3` is damage here, right? If so, can you remove it and add it to `-ii` and also remove the other one below?

```suggestion
1067.0 "Sculptor's Passion" sync / 1[56]:[^:]*:Gladiator of Sil'dih:766C:/
```